files/image_config/timezone/timezone-config.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+CURRENT_TIMEZONE=`cat /etc/timezone`
+TIMEZONE=`sonic-cfggen -d -v DEVICE_METADATA[\'localhost\'][\'timezone\']`
+
+if [ ! $TIMEZONE == "" ]
+then
+    sudo bash -c "echo $TIMEZONE > /etc/timezone"
+    sudo bash -c "dpkg-reconfigure -f noninteractive tzdata"
+    sudo bash -c "systemctl restart rsyslog"
+    sudo bash -c "systemctl restart cron"
```

Comment:
Aren't these already run as root ? Why need "sudo bash -c " ?
Could you use "timedatectl settimezone US/Pacific" to set the timezone instead of directly modifying the files ?

Reply:
I left previous company i can't modify this pull request any more.